services/management/commands/turku_service_import/units.py

```diff
@@ -194,7 +194,11 @@ def _handle_extra_info(self, obj, unit_data):
         for extra_info_data in location_data.get('lisatiedot', []):
             try:
                 koodi = extra_info_data['lisatietotyyppi'].get('koodi')
-                field_mapping = EXTRA_INFO_FIELD_MAPPING[koodi]
+                if koodi == '5':
+                    obj.picture_url = extra_info_data.get('nimi')
+                    continue
```

Comment:
Is there a reason why this isn't implemented by adding the field to `EXTRA_INFO_FIELD_MAPPING` ?

Reply:
I agree here, seems like adding
```
'6': {'nimi': 'picture_url'},
```
would do the same thing here I think :)

Reply:
`'5': {'nimi': 'picture_url'},` to be exact 😉

Reply:
Yes makes much more sense, I don't know how I missed that - now fixed!

Reply:
well, can't blame you, the thing is pretty cryptic and of course not commented. This was a bit easier for me as I'm the original author 😄